plugin-flex-ts-template-v2/.prettierignore

```diff
@@ -1 +1,2 @@
-README.md
\ No newline at end of file
+README.md
+*.md
```

Comment:
?? :)

Reply:
When I was editing markdown files prettier was wildly inflating the changeset :D

I will remove this line and resolve.